src/fund_my_watcard/main.py

```diff
@@ -128,5 +135,58 @@ def main():
         if query_yes_no(WILL_RESET_YOUR_CONFIG_FILE_WARNING, "no"):
             reset_config_file()
 
+    elif args.valid:
+        if os.path.isfile(CONFIG_FILE_PATH):
+            with open(CONFIG_FILE_PATH) as f:
+                _config = json.load(f)
+        else:
+            report_error(CAN_NOT_FIND_CONFIG_FILE)
+
+        if _config["encrypted"] == "True":
+            report_warning(CONFIG_FILE_HAS_BEEN_ENCRYPTED)
+        else:
+            if re.fullmatch('[!@#$%^&*(),.?":{}|<>]', _config["userName"]):
+                report_error("userName has special characters, please remove them.")
+
+            if re.fullmatch("([a-zA-Z]+ +)", _config["ordName"]):
+                report_error("ordName has characters other than letters, please remove them.")
+
+            if len(_config["phoneNumber"].replace(" ", "")) != 10:
```

Comment:
Do we need to consider this case: "+1 xxx xxx xxxx"?

Reply:
I don't think so since all Canadian numbers have country code +1. We could force user to use a certain phone number format or consider the case `(xxx) xxx xxxx` and `xxx-xxx-xxxx` and `(xxx)-xxx-xxxx`.